dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -196,6 +196,154 @@
+       "choices": [
+        {
+         "value": "hangouts",
+         "text": "Google Hangouts"
+        },
+        {
+         "value": "zoom",
+         "text": "Zoom"
+        },
+        {
+         "value": "teams",
+         "text": "Microsoft Teams"
+        },
+        {
+         "value": "goto_meeting",
+         "text": "GoToMeeting"
+        },
+        {
+         "value": "skype",
+         "text": "Skype"
+        },
+        {
+         "value": "dont_know",
+         "text": "Don't Know/Unsure"
+        }
+       ],
+       "hasOther": true,
+       "otherText": "Other"
+      },
+      {
+       "type": "matrix",
+       "name": "friday_institute_sync_tools",
+       "title": "Thinking about the video conferencing portions of the workshop, how much do you agree with the following?  ",
+       "columns": [
+        {
+         "value": "1",
+         "text": "Strongly Disagree"
+        },
+        {
+         "value": "2",
+         "text": "Disagree"
+        },
+        {
+         "value": "3",
+         "text": "Slightly Disagree"
+        },
+        {
+         "value": "4",
+         "text": "Neutral"
+        },
+        {
+         "value": "5",
+         "text": "Slightly Agree"
+        },
+        {
+         "value": "6",
+         "text": "Agree"
+        },
+        {
+         "value": "7",
+         "text": "Strongly Agree"
+        }
+       ],
+       "rows": [
+        {
+         "value": "smoothly_for_all",
+         "text": "Overall the video conferencing technology seemed to work smoothly for all participants."
+        },
+        {
+         "value": "personal_tech_adequate",
+         "text": "My personal technology infrastructure - internet connection, bandwidth, screen size, software - was adequate for participating in the workshop."
+        },
+        {
+         "value": "video_conf_engaging",
+         "text": "I found the video conferencing portions of the workshop highly engaging."
+        }
+       ]
+      },
+      {
+       "type": "matrix",
+       "name": "friday_institute_async_tools",
+       "title": "This workshop also included activities you completed away from the larger group made available through your workshop website in the Friday Institute's Professional Learning and Collaboration Environment (PLACE) ",
+       "columns": [
+        {
+         "value": "1",
+         "text": "Strongly Disagree"
+        },
+        {
+         "value": "2",
+         "text": "Disagree"
+        },
+        {
+         "value": "3",
+         "text": "Slightly Disagree"
+        },
+        {
+         "value": "4",
+         "text": "Neutral"
+        },
+        {
+         "value": "5",
+         "text": "Slightly Agree"
+        },
+        {
+         "value": "6",
+         "text": "Agree"
+        },
+        {
+         "value": "7",
+         "text": "Strongly Agree"
+        }
+       ],
+       "rows": [
+        {
+         "value": "moodle_worked_well_for_all",
```

Comment:
should the value be moodle or place?

Reply:
doesn't matter.  public facing name is PLACE.  But's a moodle site.  leave.